include/juice/juice.h

```diff
@@ -27,11 +27,7 @@ extern "C" {
 #include <stddef.h>
 #include <stdint.h>
 
-#ifdef _WIN32
-#define JUICE_EXPORT __declspec(dllexport)
-#else
-#define JUICE_EXPORT
-#endif
+#include "juice_export.h"
```

Comment:
The header generation is rather elegant, however it breaks the [direct Makefile build](https://github.com/paullouisageneau/libjuice#build-directly-with-make-linux-only). I'd like to keep it working, for instance by guarding the `juice_export.h` include with a define, for instance `JUICE_HAS_EXPORT_HEADER` (just a suggestiuon, there might be a better way).

Reply:
I've added the guard. It's a simple solution, but there is a slight disadvantage with this approach - the users of the library will not use `juice_export.h` unless they define `JUICE_HAS_EXPORT_HEADER` themselves.